### PR TITLE
[IN-2442] pass FASTLANE_SESSION as a b64 string

### PR DESCRIPTION
**Pull request by bitrise-tom**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/simulator-tests/tasks/main.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@
   environment:
     XCODE_INSTALL_USER: "{{ XCODE_INSTALL_USER }}"
     XCODE_INSTALL_PASSWORD: "{{ XCODE_INSTALL_PASSWORD }}"
-    FASTLANE_SESSION: "{{ FASTLANE_SESSION }}"
+    FASTLANE_SESSION: "{{ FASTLANE_SESSION | b64decode }}"
```
Comment (peterheja-bitrise) [inline roles/simulator-tests/tasks/main.yml:23]: Use `'` instead of `"`.

File: roles/simulators/tasks/install.yml [hunk 2]
```diff
@@ -10,5 +10,5 @@
   environment:
     XCODE_INSTALL_USER: "{{ XCODE_INSTALL_USER }}"
     XCODE_INSTALL_PASSWORD: "{{ XCODE_INSTALL_PASSWORD }}"
-    FASTLANE_SESSION: "{{ FASTLANE_SESSION }}"
+    FASTLANE_SESSION: "{{ FASTLANE_SESSION | b64decode }}"
```
Comment (peterheja-bitrise) [inline roles/simulators/tasks/install.yml:13]: Use `'` instead of `"`.

File: roles/simulators/tasks/main.yml [hunk 3]
```diff
@@ -23,4 +23,4 @@
   environment:
     XCODE_INSTALL_USER: "{{ XCODE_INSTALL_USER }}"
     XCODE_INSTALL_PASSWORD: "{{ XCODE_INSTALL_PASSWORD }}"
-    FASTLANE_SESSION: "{{ FASTLANE_SESSION }}"
+    FASTLANE_SESSION: "{{ FASTLANE_SESSION | b64decode }}"
```
Comment (peterheja-bitrise) [inline roles/simulators/tasks/main.yml:26]: Use `'` instead of `"`.

File: roles/xcode/tasks/main.yml [hunk 4]
```diff
@@ -10,7 +10,7 @@
   environment:
     XCODE_INSTALL_USER: "{{ XCODE_INSTALL_USER }}"
     XCODE_INSTALL_PASSWORD: "{{ XCODE_INSTALL_PASSWORD }}"
-    FASTLANE_SESSION: "{{ FASTLANE_SESSION }}"
+    FASTLANE_SESSION: "{{ FASTLANE_SESSION | b64decode }}"
```
Comment (peterheja-bitrise) [inline roles/xcode/tasks/main.yml:13]: Use `'` instead of `"`.

**Review by peterheja-bitrise — CHANGES_REQUESTED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
lgtm

**Review by fehersanyi-bitrise — APPROVED** [review]

**Pull request merged**